### PR TITLE
Updates support for protobuf to match pending network upgrades

### LINKED ISSUE #138: Check version of protobuf in pom.xml
Check the version of the protobuf that's imported through the pom.xml to make sure it matches with mainnet/testnet.

### PR DESCRIPTION
**Pull request by gregscullard**
**Detailed description:**
Updates the protobuf library used by mirror node to match testnet/mainnet upgrade version.
-A number of new ResponseCodes are added with the upgrade, mirror node needs to support them.
-Added error handling around unknown response codes to avoid NPE

**Which issue(s) this PR fixes:**
Fixes #138

**Checklist**
- [ ] Documentation added
- [X] Tests updated

### CODE REVIEW

File: src/main/java/com/hedera/recordFileLogger/RecordFileLogger.java [hunk 1]
```diff
@@ -328,7 +328,13 @@ public static boolean storeRecord(long counter, Instant consensusTimeStamp, Tran
 			sqlInsertTransaction.setLong(F_TRANSACTION.FK_PAYER_ACCOUNT_ID.ordinal(), fkPayerAccountId);
 
 			long fk_result_id = -1;
-			String responseCode = ResponseCodeEnum.forNumber(txRecord.getReceipt().getStatus().getNumber()).getValueDescriptor().getName();
+			String responseCode = "";
+			try {
+				responseCode = ResponseCodeEnum.forNumber(txRecord.getReceipt().getStatus().getNumber()).getValueDescriptor().getName();
+			} catch (Exception e) {
+				log.error("Unknown response code, com.hederahashgraph.api library likely out of date. Exception {}", e );
```
Comment (mike-burrage-hedera) [inline src/main/java/com/hedera/recordFileLogger/RecordFileLogger.java]: I'd say make this a warning. We should raise the notice and do something about it (eventually), but it's not immediately blocking.

Comment (gregscullard) [reply src/main/java/com/hedera/recordFileLogger/RecordFileLogger.java]: The database contains a "not  known to db" so we could log that against the transaction and continue, while issuing a warning ?

Comment (gregscullard) [reply src/main/java/com/hedera/recordFileLogger/RecordFileLogger.java]: Although I'm uncomfortable leaving a transaction with an unknown status in the database, some business logic might depend on it, e.g. did a crypto transfer occur or not ?
Not knowing for sure could be problematic which is why I went for error and fail.

Comment (mike-burrage-hedera) [reply src/main/java/com/hedera/recordFileLogger/RecordFileLogger.java]: Makes sense. I'll add a ticket for some changes to this logic in the next release.

**Pull request merged**